### ISSUE TITLE
error "missing path" when trying to build sfx bundle with css plugin

### ISSUE THREAD

**Issue by yoavniran** [systemjs/plugin-css#47]
Hi,
Im using jspm version: 0.16.0-beta.3 
with an extremely simple test project:

my src/app.js looks like this:

```js
require("styles/myTest.css!");

var tools = require("./tools");

tools.doSomething();
```

Everything works in dev and if I use: _jspm bundle src/app target/app-bundle.js_

however, when I try:_jspm bundle**-sfx** src/app target/app-bundle.js_
I get this error: "missing path"

err  AssertionError: missing path
         at Module.require (module.js:363:3)
         at require (module.js:384:17)
         at c:\Dropbox\Projects\jtv\tests\systemjs\node_modules\jspm\node_modules\systemjs-builder\lib\compile.js:157:16

Its failing on line 
```js
 compiler = require(compilerMap[format]);
```
with format = "defined". which as far as i understand, it should not be trying to do since intentionally there isnt a compiler for that type.

Im guessing im missing some configuration or something?
 thanks.

**Comment by guybedford** [comment]
I believe this has been fixed on SystemJS master actually - hoping to post out a release very soon.

**Comment by yoavniran** [comment]
hi @guybedford thats great to hear,
can you point me to the code fix? Im doing a POC now and its an important part for me to prove working.

thanks!

**Comment by guybedford** [comment]
Actually it should already be working in SystemJS builder 0.12.2. Are you definitely running this version?

**Comment by guybedford** [comment]
Sorry, I mean 0.12.1. But I see the issue, posting a fix now.

**Comment by guybedford** [comment]
Fixed in builder 0.12.2.

**Comment by yoavniran** [comment]
works! thanks a lot!!!
